manifests/nodejs.yml

```diff
@@ -819,49 +1063,37 @@ tests/:
       Test_Debugger_Exception_Replay: missing_feature (feature not implented)
     test_debugger_expression_language.py:
       Test_Debugger_Expression_Language:
-        '*': irrelevant
-        express4: v5.48.0
-        express4-typescript: v5.48.0
-        uds-express4: v5.48.0
+        '*': *ref_5_48_0
+        nextjs: irrelevant
```

Comment:
This also enables the `express5` tests. Not necessarily an issue, but just thought I'd mention it since it's not mentioned in the PR description. I didn't add this originally because I didn't think there would be much gained by also running those tests. But if we want to be certain, it's ok with me (the same goes for all the other `test_debugger-*` tests below)